experimental/cloudstrapper/playbooks/roles/control/tasks/patch-terraform-v1.3.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: copy aws eks.tf
+  copy:
+    src: "roles/control/files/{{ orc8rVersion }}/orc8r-aws/eks.tf"
```

Comment:
I see that we are defining 1.3/1.4 version here. Given that we are porting to Magma Repo. Should we just build this on current version ?